Flaky test `TestBatchServiceCreate`

Test `TestBatchServiceCreate` appears to be flaky. [CI example](https://github.com/ethersphere/bee/actions/runs/4271721602/jobs/7436386800)

```
 === RUN   TestBatchServiceCreate
=== RUN   TestBatchServiceCreate/expect_put_create_put_error
    batchservice_test.go:134: expected error
=== RUN   TestBatchServiceCreate/passes
=== RUN   TestBatchServiceCreate/passes_without_recovery
=== RUN   TestBatchServiceCreate/batch_with_near-zero_val_rejected
--- FAIL: TestBatchServiceCreate (0.00s)
    --- FAIL: TestBatchServiceCreate/expect_put_create_put_error (0.00s)
    --- PASS: TestBatchServiceCreate/passes (0.00s)
    --- PASS: TestBatchServiceCreate/passes_without_recovery (0.00s)
    --- PASS: TestBatchServiceCreate/batch_with_near-zero_val_rejected (0.00s)
 ```

--- a/pkg/postage/batchservice/batchservice_test.go
+++ b/pkg/postage/batchservice/batchservice_test.go
@@ -62,7 +62,7 @@ func (m *mockBatchListener) HandleDepthIncrease(_ []byte, _ uint8) {
 	m.diluteCount++
 }
 
-func TestBatchServiceCreate(t *testing.T) {
+func TestBatchServiceCreate_FLAKY(t *testing.T) {
 	testChainState := postagetesting.NewChainState()
 
 	validateNoBatch := func(t *testing.T, testBatch *postage.Batch, st *mock.BatchStore) {

